README.md

```diff
@@ -49,13 +50,21 @@
   - [dangerjs](https://github.com/danger/danger/blob/master/Dangerfile)
   - [editorconfig](https://editorconfig.org)
 
+
+## Deployment
```

Comment:
We deploy on every PR or only when that PR is merged into the target branch?

Reply:
this is for **Manual Deployment**
I'll move this out 👍